conductr_cli/conduct_info_list.py

```diff
@@ -38,7 +38,7 @@ def display_bundles_default(args, is_license_success, conductr_license, bundles)
     data.insert(0, {
         'id': 'ID',
         'name': 'NAME',
-        'compatibility_version': 'VER',
+        'tag': 'TAG',
```

Comment:
I think it would be more accurate if we show `VER` as the header if `tags` is missing for every bundle in the HTTP response -- in that case, we are in fact showing `compatabilityVersion` so it would be nice if the headers reflect that. wdyt?